File: Sample/java/src/main/java/net/nend/sample/java/banner/layoutpatterns/LayoutSampleActivity.java
package net.nend.sample.java.banner.layoutpatterns;

import android.os.Bundle;

import net.nend.sample.java.R;
import net.nend.sample.java.SimpleListActivity;

import java.util.ArrayList;
import java.util.List;

public class LayoutSampleActivity extends SimpleListActivity {

    private static final List<String> SAMPLE_ACTIVITIES = new ArrayList<String>() {
        {
            add(TopActivity.class.getName());
            add(BottomActivity.class.getName());
            add(AttachAndDetachActivity.class.getName());
            add(ListViewActivity.class.getName());
            add(ViewPagerActivity.class.getName());
            add(FragmentReplaceActivity.class.getName());
        }
    };

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        instantiateMenuListFragment(R.layout.layout_sample, SAMPLE_ACTIVITIES);
    }
}

File: Sample/kotlin/src/main/java/net/nend/sample/kotlin/banner/layoutpatterns/LayoutSampleActivity.kt
package net.nend.sample.kotlin.banner.layoutpatterns

import android.os.Bundle
import net.nend.sample.kotlin.R
import net.nend.sample.kotlin.SimpleListActivity
import java.util.*

class LayoutSampleActivity : SimpleListActivity() {

    private val list: ArrayList<String> = object : ArrayList<String>() {
        init {
            add(TopActivity::class.java.name)
            add(BottomActivity::class.java.name)
            add(AttachAndDetachActivity::class.java.name)
            add(ListViewActivity::class.java.name)
            add(ViewPagerActivity::class.java.name)
            add(FragmentReplaceActivity::class.java.name)
        }
    }

    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        instantiateMenuListFragment(R.layout.layout_sample, list)
    }
}